### DISCUSSION TITLE
Report errors here! Help us perfect all ten test sets!

### DISCUSSION THREAD

**Discussion by cgnorthcutt** [cgnorthcutt/label-errors#4] [General]
If you find any errors in the released corrected test sets, or on https://labelerrors.com, please collect all the errors and report them here in bulk.

If you agree with someone else's reporting of errors, please thumbs up their post!

**Comment by Symbiomatrix** [comment]
Dataset: Quickdraw
ID: 7249992
Actual: Brush
(The drawer clearly marked it as one, though I think it's not actually amidst the set's labels.)

ID: 47567427
Actual: Snarky encircled text

ID: 8728702
Actual: Candle
(Label is correct, top was cut off but it's no fork.)

ID: 13030287
Actual: Diamond
(The shape variety; poorly drawn as are some samples but label is correct.)

ID: 27798013
Actual: Microphone
(The two are often interchangeable in less detailed drawings, but what is somewhat consistent is the handle to head ratio, smaller for microphones.)

**Comment by cgnorthcutt** [comment]
thanks!

Make it a great day,

Curtis

--


Curtis Northcutt
Cleanlab
<https://d4GkP704.na1.hs-sales-engage.com/Ctc/I9+23284/d4GkP704/Jks2-6qcW69sMD-6lZ3nkW2GsM__7rYkwrW7DrTS74rwmVkW2TcRQD1V1LzhVNH-Pm3Z6MJTW4KvZzp8KM06wW6-qJNN2qd2VBW4ZKksP4-2DZ9W5YbwCr6xQXnJW4GMcP258BDWNW8CcQ2C11sxwxN4K_sdqXK3GfW8Y-Vs329png0W3dcmrG3x-M31N8RWJKB20wX5W3R5L3p276zNlW3Kv2yR3TBJ02N7d1ftx5HFt4VB0k4n8n62pBN2_WgthwRp--W37Qy7f85rk7sf2lX90404>
|
CEO & Co-Founder

This email and any attachments thereto may contain private, confidential,
and privileged material for the sole use of the intended recipient. Any
review, copying, or distribution of this email (or any attachments thereto)
by others is strictly prohibited. If you are not the intended recipient,
please contact the sender immediately and permanently delete the original
and any copies of this email and any attachments thereto.

**Comment by geangs** [comment]
Dataset: ImageNet
ID: 00020256
Label: Red wolf
Correction: Maned wolf
https://en.wikipedia.org/wiki/Maned_wolf
